src/main/java/de/tsystems/mms/apm/performancesignature/util/PerfSigUtils.java

```diff
@@ -145,13 +144,8 @@ public static Area calcDefaultSize() {
     }
 
     public static UsernamePasswordCredentials getCredentials(final String credsId) {
```

Comment:
i used `Stapler.getCurrentRequest().findAncestorObject(Item.class)` to get the required item, but its always null